platipy/imaging/visualisation/comparison.py

```diff
@@ -48,7 +48,7 @@ def contour_comparison(
     subsubtitle="",
     contour_cmap=plt.cm.get_cmap("rainbow"),
     structure_name_dict=None,
-    img_vis_kw={},
+    img_vis_kw=None,
```

Comment:
@rnfinnegan @dalmouiee, I just discovered this bug... Remember in pydicer how we discussed the pylint warning not to set a list or dictionary as a default value? Well, I just uncovered a bug here which was exactly due to this. If using {} as a default value here, a value (`cut`) is set in the dict later in the code. Since I call this function in a loop in my scripts, that value is set only on the first iteration (where as it should be set fresh in each loop iteration). So by setting the default to None and then initalising the empty dict in the code this bug was fixed.

A great use case to demonstrate why pylint warnings are so useful. And we definitely need to start being more vigilant in checking platipy's code with pylint! :D

Reply:
Awesome yeah will be great to apply pylint on the codebase! I guess it'll have to be a gradual process since it won't be straightforward on something this large and comlpex?

Reply:
Yeah for sure, definitely a lot more straightforward to ensure pylint converage from the start. But yes for platipy Rob and I can start to ensure pylint compliance file by file :)